taichi/backends/wasm/codegen_wasm.cpp

```diff
@@ -173,14 +173,14 @@ class CodeGenLLVMWASM : public CodeGenLLVM {
     ir->accept(this);
     finalize_taichi_kernel_function();
 
-    auto wasm_materialize_name = "wasm_materialize";
-
     // compile_module_to_executable
     // only keep the current func
     TaichiLLVMContext::eliminate_unused_functions(
         module.get(), [&](std::string func_name) {
-          return offloaded_task_name == func_name ||
-                 wasm_materialize_name == func_name;
+          return func_name == offloaded_task_name ||
+                 func_name == "wasm_materialize" ||
```

Comment:
nit: Store these function names into a `constexpr std::array` so that we can reuse?